paper/references.bib

```diff
@@ -914,3 +914,60 @@ @inproceedings{nieuwenhuis2005proof
   year={2005},
   organization={Springer}
 }
+
+@book{Goodfellow-et-al-2016,
+    title={Deep Learning},
+    author={Ian Goodfellow and Yoshua Bengio and Aaron Courville},
+    publisher={MIT Press},
+    doi= {10.1017/CBO9781139172752},
```

Comment:
Is this right?  Isn't this Term Rewriting?

https://www.cambridge.org/core/books/term-rewriting-and-all-that/71768055278D0DEF4FFC74722DE0D707